PocketKit/Sources/PocketKit/MyList/SearchItemsList/SearchViewModel.swift

```diff
@@ -86,6 +91,25 @@ class SearchViewModel: ObservableObject {
         return searches
     }
 
+    public func searchItems(with searchTerm: String) -> [SavedItem] {
+        let term = searchTerm.trimmingCharacters(in: .whitespaces).lowercased()
+        var listOfItems = [SavedItem]()
+        guard isPremium else {
+            guard let list = source.searchSaves(search: term), list.isEmpty else {
+                listOfItems = []
+                return listOfItems
+            }
+            listOfItems = list
+            return listOfItems
```

Comment:
This codeblock seems to be a duplicate of the immediately following `guard let list = ...`. Do we still need to guard against `isPremium`, since that's now in `Source`? I feel like there's an opportunity to remove some duplicate code.

Reply:
yup yup. cleaned it up.